main.py

```
@@ -1,44 +1,87 @@
+
+# Простая база "платных" пользователей
+paid_users = set()
+
+# ---------------- START ----------------
+@bot.message_handler(commands=['start'])
+def start(message):
+    markup = types.ReplyKeyboardMarkup(resize_keyboard=True)
+
+    btn1 = types.KeyboardButton("📥 Скачать видео")
+    btn2 = types.KeyboardButton("💰 Купить доступ")
+    btn3 = types.KeyboardButton("ℹ️ Помощь")
```

Ruff (0.15.9)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)